server/index.js

```diff
@@ -1,18 +1,38 @@
 const express = require('express');
 const cors = require('cors')
+const petModel = require('./db/index');
+const mongoose = require('mongoose');
 
 const app = express();
 const port = 8000;
 
 app.use(cors())
 app.use(express.json())
 
+mongoose.connect('mongodb://localhost:27017/pet-cards', () => console.log('Connected to DB'))
+
 const petList = [];
 
 app.post('/api/pet', (req, res) => {
```

Comment:
Please do api/v1 like we did in chat app

Reply:
Done